src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -401,8 +401,10 @@ void AresDnsResolver::MaybeStartResolvingLocked() {
       // new closure API is done, find a way to track this ref with the timer
       // callback as part of the type system.
       Ref(DEBUG_LOCATION, "next_resolution_timer_cooldown").release();
-      grpc_timer_init(&next_resolution_timer_, ms_until_next_resolution,
-                      &on_next_resolution_);
+      grpc_timer_init(
+          &next_resolution_timer_,
+          grpc_core::ExecCtx::Get()->Now() + ms_until_next_resolution,
```

Comment:
No need for `grpc_core::` prefix here, since we're already inside of that namespace.

Reply:
done